posthog/tasks/exporter.py

```diff
@@ -37,23 +39,21 @@
 
 # export_asset is used in chords/groups and so must not ignore its results
 @shared_task(
-    autoretry_for=(Exception,),
-    max_retries=5,
```

Comment:
Wow, this might answer why exports sometimes take unbelievably long and still fail.

Reply:
I actually think they have no clear "now it's failed" state... so this doesn't make them take longer, just try needlessly. To mark hard fails would be another todo, I'll see if there is time to do that.